packages/liveblocks-react/src/index.ts

```diff
@@ -15,3 +15,6 @@ export type {
 // Re-exports from @liveblocks/client, for convenience
 export type { Json, JsonObject } from "@liveblocks/client";
 export { shallow } from "@liveblocks/client";
+
+// Re-exports from @liveblocks/core, for convenience
+export type { ThreadsFilterOptions } from "@liveblocks/core";
```

Comment:
We re-export `ThreadsFilterOptions` from `@liveblocks/core`. Want to confirm if this is okay and if we are okay with this name. @marcbouchenoire @GuillaumeSalles

Reply:
I didn't know you had a follow-up PR, [I renamed it to `ThreadsOptions`](https://github.com/liveblocks/liveblocks/pull/1351/commits/0735a9e6674b2c5768cd5bb97d4a05453827987a) in the `comments-unread` branch since it will include other options over time but it's a private type so it's not super important.